lib/gui/app/modules/image-writer.js

```diff
@@ -327,7 +346,14 @@ exports.flash = (image, drives) => {
 
     // eslint-disable-next-line no-magic-numbers
     if (drives.length > 1) {
-      analytics.logEvent('Write failed', analyticsData)
+      const { results } = flashState.getFlashResults()
+      const event = _.assign({
+        errors: results.errors,
+        devices: results.devices
+      },
+      analyticsData,
+      { status: 'failed' })
```

Comment:
Why explicitly add `{ status: 'failed' }`? Is this something that's left over from development?

Reply:
It's to make filtering easier on Mixpanel